tests/providers/google/cloud/operators/test_kubernetes_engine.py

```diff
@@ -182,6 +182,19 @@ def test_execute(self, file_mock, mock_execute_in_subprocess, mock_gcp_hook, exe
 
         assert self.gke_op.config_file == FILE_NAME
 
+    def test_config_file_throws_error(self):
+        with pytest.raises(AirflowException):
+            GKEStartPodOperator(
+                project_id=TEST_GCP_PROJECT_ID,
+                location=PROJECT_LOCATION,
+                cluster_name=CLUSTER_NAME,
+                task_id=PROJECT_TASK_ID,
+                name=TASK_NAME,
+                namespace=NAMESPACE,
+                image=IMAGE,
+                config_file="/path/to/alternative/kubeconfig"
```

Comment:
```suggestion
                config_file="/path/to/alternative/kubeconfig",
```

Should fix failing static checks: https://github.com/apache/airflow/pull/17700/checks?check_run_id=3366530442#step:10:185